include/client.h

```diff
@@ -31,22 +31,58 @@ namespace mixer_client {
 // is completed.
 using DoneFunc = std::function<void(const ::google::protobuf::util::Status&)>;
 
+// Define transport context status
+typedef enum {
+  CTX_OK = 0,
+  CTX_NOT_EXIST,
+} CtxStatus;
+
+// Transport Context ID type.
+typedef int CtxID;
```

Comment:
Here the Ctx really means streaming in the transport layer.   It is NOT attribute_ctx.